### PR TITLE
Build addons-actions with ts-up

### PR DESCRIPTION
**Pull request by Saunved**
Issue: https://github.com/storybookjs/storybook/issues/18732

## What I did
Followed the instructions from the above issue. Not very sure about the 'preview' setup, please let me know if it needs fixing.
The example was working correctly locally.

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots? Will be tested with CI
- [ ] Does this need a new example in the kitchen sink apps? No
- [ ] Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: addons/actions/preview.js [hunk 1]
```diff
@@ -1 +1 @@
-export * from './dist/esm/preset/preview';
+import './dist/preset/preview';
```
Comment (ndelangen) [inline addons/actions/preview.js:1]:
```suggestion
export * from './dist/preset/preview';
```

File: addons/actions/package.json [hunk 2]
```diff
@@ -73,6 +92,13 @@
   "publishConfig": {
     "access": "public"
   },
+  "bundler": {
+    "entries": [
+      "./src/index.ts",
+      "./src/manager.tsx",
+      "./src/preset/preview.tsx"
```
Comment (ndelangen) [inline addons/actions/package.json:99]: Please do not have a nested directory structure in entries.

Entrypoints should be directly in the `src` directory.

**Comment by Saunved** [comment]
@ndelangen I am stuck on this -- not sure what could fix this failing test or what could be wrong with the exports

**Comment by Saunved** [comment]
@ndelangen I am stuck on this PR. I don't know why the test is failing. I tried to run the cra-essentials app locally with the `next` branch as well as this branch, and both yield the same results (and the same errors). Can you please help me out?

**Comment by ndelangen** [comment]
There seems to be an issue with this change:
https://app.circleci.com/pipelines/github/storybookjs/storybook?branch=feat%2Faddons-actions-tsup

**Comment by ndelangen** [comment]
Looks like my fix didn't do anything to the CI failure. I'm not sure what's causing it.
It seems the manager part breaks throws as soon as the panel is opened.

**Pull request merged**